imports/startup/server/fixtures.js

```diff
@@ -707,3 +708,7 @@ if (Charts.find().count() === 0) {
     slug: "music-charts-2017-09-25"
   });
 }
+
+if (Parties.find().count() === 0) {
+  
```

Comment:
Just wondering what this might be. Is this the future site of some party fixtures?

Reply:
Hit and sunk.